iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -62,47 +63,40 @@ class HomeViewController: UIViewController {
 
   @IBAction private func showOtherTheme(_ sender: Any) {
     customizeNavigationBarAppearance(backgroundColor: UIColor(red: 23 / 255, green: 32 / 255, blue: 30 / 255, alpha: 1),
-                                       foregroundColor: .green)
+                                     foregroundColor: .green)
     let viewController = Factory.getOtherPaymentViewController { [weak self] result in
       self?.handleTokenResponse(with: result)
     }
     navigationController?.pushViewController(viewController, animated: true)
   }
 
   @IBAction private func getApplePayData(_ sender: Any) {
-    // Use example Apple Pay payment data.
-    guard let paymentDataURL = Bundle.main.url( forResource: "example_apple_pay_payment_data", withExtension: "json") else {
-      print("Unable to get URL of Apple Pay payment data.")
+    let paymentNetworks: [PKPaymentNetwork] = [.amex, .discover, .masterCard, .visa]
+    let paymentItem = PKPaymentSummaryItem(label: "Test item", amount: NSDecimalNumber(value: 12.99))
+    guard PKPaymentAuthorizationViewController.canMakePayments(usingNetworks: paymentNetworks) else {
+      showAlert(with: "Unable to make Apple Pay transaction.")
       return
     }
-
-    let paymentData: Data
-
-    do {
-      paymentData = try Data(contentsOf: paymentDataURL)
-    } catch {
-      print(error.localizedDescription)
+    let request = PKPaymentRequest()
+    request.currencyCode = "USD"
+    request.countryCode = "US"
+    request.merchantIdentifier = "com.checkout.merchant.io.test.ios"
+    request.merchantCapabilities = PKMerchantCapability.capability3DS
+    request.supportedNetworks = paymentNetworks
+    request.paymentSummaryItems = [paymentItem]
```

Comment:
When moving this code to the new helper object, it would be nice if we could further describe things for consumer
```
/**
    For complete information on the payment request customisation,
        see oficial docs at https://developer.apple.com/documentation/passkit/pkpaymentrequest
*/
// Payment object that Apple will generate request with
let request = PKPaymentRequest()

// Three-letter code for the currency used by this payment request
request.currencyCode = "USD"

.....

// This value must match one of the merchant identifiers specified in the app’s entitlements
request.merchantIdentifier = "com.checkout.merchant.io.test.ios"

....
```

This code is not just executable code in isolation, its in effect our documentation and our knowledge of the domain, helping someone advance quicker to integration. 
Otherwise, we should not include this code at all, and let consumer find out naturally via Apple's own docs!